code/modules/projectiles/magazines/pistols.dm

```diff
@@ -108,7 +108,7 @@
 /obj/item/ammo_magazine/pistol/holdout
 	name = "tiny pistol magazine (.22)"
 	desc = "A surprisingly small magazine, holding .22 bullets. No Kolibri, but it's getting there."
-	default_ammo = /datum/ammo/bullet/pistol/tiny
+	default_ammo = /datum/ammo/bullet/pistol
```

Comment:
Is this a buff to the holdout pistol?

Reply:
nope pistol/tiny is actually the exact same as pistol from what i saw except in name